org.kde.gwenview.json

```diff
@@ -257,11 +257,17 @@
             "name": "kImageAnnotator",
             "buildsystem": "cmake-ninja",
             "builddir": true,
+            "post-install": [
+                "install -d -m 755 $FLATPAK_DEST/usr/lib/cmake/kImageAnnotator",
```

Comment:
I think you should drop `/usr` from the path

Reply:
Oh, good catch, my bad